dependency/vault_list_test.go

```diff
@@ -68,13 +68,16 @@ func TestNewVaultListQuery(t *testing.T) {
 func TestVaultListQuery_Fetch(t *testing.T) {
 	t.Parallel()
 
-	clients, vault := testVaultServer(t)
-	defer vault.Stop()
+	clients, vault := testVaultServer("listfetch", "1")
+	secrets_path := vault.secrets_path
```

Comment:
snake case -> camel case

Reply:
Changing the struct variable to camelCase... leaving local alone.